front/pages/api/v1/data_sources/[user]/[name]/search.ts

```diff
@@ -3,9 +3,9 @@ import { auth_api_user } from "@app/lib/api/auth";
 import { APIError } from "@app/lib/api/error";
 import { JSONSchemaType } from "ajv";
 import { User, DataSource, Provider } from "@app/lib/models";
-import { Op } from "sequelize";
 import { credentialsFromProviders } from "@app/lib/providers";
 import { parse_payload } from "@app/lib/http_utils";
+import { DocumentType } from "@app/lib/types";
```

Comment:
Note that `Document` was not imported and was referring to something completely random :)